src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -56,3 +57,8 @@ def get(self, svc_id):
             'osd_metadata': mgr.get_metadata('osd', svc_id),
             'histogram': histogram,
         }
+
+    @RESTController.Resource('POST', query_params=['deep'])
+    def scrub(self, svc_id, deep=False):
+        api_scrub = "osd deep-scrub" if deep else "osd scrub"
```

Comment:
I don't think anyone is actually converting `deep` into a proper Boolean. Can you confirm that `POST ...0/scrub?deep=False` doesn't trigger a deep scrub?

Reply:
Done.